dali/operators/image/crop/bbox_crop.cc

```diff
@@ -96,191 +96,232 @@ struct Range {
-index of the coordinate.
-Coordinates are relative to the original image dimensions (i.e. range [0.0, 1.0]), representing
-either ``start`` and ``end`` of the region or ``start`` and shape` depending on the value of ``bbox_layout``.
-E.g. ``bbox_layout="xyXY"`` means the bounding box coordinates are following the order
-``start_x, start_y, end_x, end_y``, while ``bbox_layout="xyWH"`` indicates the order is
-``start_x, start_y, width, height``. See more in ``bbox_layout`` argument description.
-
-A second input ``labels`` can be optionally provided, representing the labels associated with each of
-the bounding boxes.
-
-Outputs: 0:``anchor``, 1:``shape``, 2:``bboxes``, (3:``labels``,)
-
-The resulting crop parameters are provided as two separate outputs ``anchor`` and
-``shape``, that can be fed directly to ``Slice`` operator to perform the actual cropping of the
-original image. ``anchor`` and ``shape`` contain starting coordinates and dimensions for the crop, as
-``[x, y, (z,)]`` and ``[w, h, (d,)]`` respectively. The coordinates can be represented in absolute or
-relative terms, depending of whether the fixed ``crop_shape`` was used or not.
-
-The third and fourth output correspond to the adjusted bounding boxes and optionally
-their corresponding labels. Bounding boxes are always specified in relative coordinates.)code")
+the bounding boxes, and optional labels, consistent.
+
+This means that after applying the random crop operator to the image coordinate space, the bounding
+boxes will be adjusted or filtered out to match the cropped ROI. The applied random crop operation
+is constrained by the arguments that are provided to the operator.
+
+The cropping window candidates are randomly selected until one matches the overlap restrictions
+that are specified by the ``thresholds`` argument. ``Thresholds`` values represent a minimum overlap
+metric that are specified by threshold_type, such as the intersection-over-union of the cropping
+window and the bounding boxes or the relative overlap as a ratio of the intersection area and
+the bounding box area.
+
+Alternatively, you can prevent crop as one of the valid outcomes of the random process by
+using ``allow_no_crop``.
+
+The following modes of a random crop mopes are available:
+
+- | Randomly shaped window, which is randomly placed in the original input space.
+  | The random crop window dimensions are selected based on the provided aspect_ratio and relative
+    area restrictions.
+- | Fixed size window, which is randomly placed in the original input space.
+  | The random crop window dimensions are taken from the ``crop_shape`` argument and the anchor is
+  | selected randomly selected.
+  | When providing ``crop_shape``, a second argument, ``input_shape``, should
+    be provided that specifies the original image dimensions.
+
+  .. note::
+     These dimensions are required to scale the output bounding boxes.
```

Comment:
This note only applies to ``crop_shape`` - I think that the formatting will be off and we'll end up with a misplaced note.

Reply:
It is fine:
![image](https://user-images.githubusercontent.com/39967756/92254919-e7a2c080-eed1-11ea-8f44-72b35063bfce.png)